scripts/go-gen-asgt.sh

```diff
@@ -19,20 +17,31 @@ protoc -I./deps/googleapis -I./deps/tensorflow -I./deps/protoc-gen-openapiv2 -I.
 protoc -I./deps/googleapis -I./deps/tensorflow -I./deps/protoc-gen-openapiv2 -I./proto \
     --openapiv2_out gen/openapiv2/ \
     --openapiv2_opt logtostderr=true \
+    --openapiv2_opt allow_merge=true \
+    --openapiv2_opt merge_file_name=asgt/v2alpha.json \
```

Comment:
what's this for?

Reply:
@michalderdak So without this undocumented feature we get one swagger json file per proto file, which means we would need one Swagger UI per service. This merges all documentation for the selected proto in a single file.